docs-client/src/containers/App/index.tsx

```diff
@@ -473,6 +473,7 @@ const App: React.FunctionComponent<Props> = (props) => {
         });
         setSpecification(initialSpecification);
       } catch (e) {
+        console.log(e);
```

Comment:
```suggestion
```

Reply:
Ah sorry - it's not related to the PR title but I listed in Modifications, I'd recommend having this log or the error is completely squashed. But if still better to remove will do it